lib/rubocop/cop/metrics/abc_size.rb

```diff
@@ -7,6 +7,27 @@ module Metrics
       # configured maximum. The ABC size is based on assignments, branches
       # (method calls), and conditions. See http://c2.com/cgi/wiki?AbcMetric
       # and https://en.wikipedia.org/wiki/ABC_Software_Metric.
+      #
+      # You can have repeated "attributes" calls count as a single "branch".
+      # For this purpose, attributes are any method with no argument; no attempt
```

Comment:
and an implicit/current object receiver I guess.

Reply:
Not necessarily:
```ruby
foo.bar.baz # => 3 branches
foo.bar.qux # => 1 more branch only (`foo` and `bar` are "repeated")

n = 42
n.to_s # => 1 branch
n.to_s.to_i # => just 1 more (`to_s` "repeated")
```